lib/emque/producing/version.rb

```diff
@@ -1,5 +1,5 @@
 module Emque
   module Producing
-    VERSION = "1.3.0"
+    VERSION = "1.4.0"
```

Comment:
What do you think about a patch bump to 1.3.1 for a bug fix? Apologies  I think I said minor in the original comment.

Reply:
Oh no problem! I think that makes more sense. :)